buildSrc/src/main/java/Config.kt

```diff
@@ -1,14 +1,16 @@
 import java.math.BigDecimal
 
 object Config {
-    val kotlinVersion = "1.5.31"
```

Comment:
We alos need this to support compose latest version. @marandaneto do you see any problems with bumping this for the whole monorepo? Kotlin now is at 1.7.